libse/SubtitleFormats/SubRip.cs

```diff
@@ -252,11 +252,11 @@ private bool TryReadTimeCodesLine(string line, Paragraph paragraph)
             string[] tokens = line.Replace("-->", ":").Replace(" ", string.Empty).Split(':', ',');
             try
             {
-                int startHours = int.Parse(tokens[0]);
+                int startHours = Math.Abs(int.Parse(tokens[0]));
                 int startMinutes = int.Parse(tokens[1]);
                 int startSeconds = int.Parse(tokens[2]);
                 int startMilliseconds = int.Parse(tokens[3]);
-                int endHours = int.Parse(tokens[4]);
+                int endHours = Math.Abs(int.Parse(tokens[4]));

```

Comment:
If we don't do this we will lose times inside `new TimeCode` in line https://github.com/SubtitleEdit/subtitleedit/pull/2112/commits/388dcee6056e7e95cfdfa40017b464b150fd79bf#diff-9cc73af07ea827ffb1e7e480e3f644abR269